unidist/core/backends/mpi/core/communication.py

```diff
@@ -584,91 +621,6 @@ def recv_simple_operation(comm, source_rank):
     return comm.recv(source=source_rank)
 
 
-def send_operation_data(comm, operation_data, dest_rank, is_serialized=False):
-    """
-    Send data that consists of different user provided complex types, lambdas and buffers.
-
-    The data is serialized with ``unidist.core.backends.mpi.core.ComplexDataSerializer``.
-    Function works with already serialized data.
-
-    Parameters
-    ----------
-    comm : object
-        MPI communicator object.
-    operation_data : object
-        Data object to send.
-    dest_rank : int
-        Target MPI process to transfer data.
-    is_serialized : bool
-        `operation_data` is already serialized or not.
-
-    Returns
-    -------
-    dict or None
-        Serialization data for caching purpose or nothing.
-
-    Notes
-    -----
-    Function returns ``None`` if `operation_data` is already serialized,
-    otherwise ``dict`` containing data serialized in this function.
-    """
-    if is_serialized:
-        # Send already serialized data
-        s_data = operation_data["s_data"]
-        raw_buffers = operation_data["raw_buffers"]
-        buffer_count = operation_data["buffer_count"]
-        _send_complex_data_impl(comm, s_data, raw_buffers, buffer_count, dest_rank)
-        return None
-    else:
-        # Serialize and send the data
-        s_data, raw_buffers, buffer_count = send_complex_data(
-            comm, operation_data, dest_rank
-        )
-        return {
-            "s_data": s_data,
-            "raw_buffers": raw_buffers,
-            "buffer_count": buffer_count,
-        }
-
-
-def send_operation(
```

Comment:
Why is this method removed?

Reply:
This function has the same functionality as the send_complex_operation function. I replaced all usage and this function is no longer used.  I think this function should be removed to avoid duplication.